hil/cli.py

```diff
@@ -288,570 +190,352 @@ def serve_networks():
             pass
         sleep(sleep_time)
 
+#################################
+### NODE SUBCOMMANDS GO HERE! ###
+#################################
```

Comment:
Big comments like these that basically make sections in the file suggest to me that maybe we should just split up the file into multiple modules.

Reply:
good point. I'll make a folder for cli and make multiple modules. Will probably help with reviewing too.